test/base/SpecBase.scala

```diff
@@ -70,6 +71,11 @@ trait SpecBase
   val consigneeAddress: UkAddress = UkAddress("buildingAndStreet", "city", "NE99 1XN")
   val configKey                   = "config"
 
+  val officeOfDestination: CustomsOffice            = new CustomsOffice("ABC12345", Some("Test"), Some("+44 7760663422"))
+  val officeOfDestinationNoTel: CustomsOffice       = new CustomsOffice("ABC12345", Some("Test"), None)
+  val officeOfDestinationNoNameTel: CustomsOffice   = new CustomsOffice("ABC12345", None, Some("+44 7760663422"))
+  val officeOfDestinationNoNameNoTel: CustomsOffice = new CustomsOffice("ABC12345", None, None)
```

Comment:
Can these just be moved to the view spec?

Reply:
Can do.